lib/spack/spack/test/build_environment.py

```diff
@@ -111,7 +101,7 @@ def _set_wrong_cc(x):
 
 
```

Comment:
Now that the logic for finding include dirs and lib dirs is moved into build_environment, please add a test to confirm that build environment emits the directories we expect.

Reply:
https://github.com/spack/spack/pull/8136/commits/37ced1fb11cd19a45f1f8ae44f85ae30fb65675d added `test_set_build_environment_variables` - is that what you were looking for?